### PR TITLE
Updated websites.csv with new URLs

### PR DESCRIPTION
**Pull request by ChrisBratti-VA**

### CODE REVIEW

File: facilities/src/main/resources/websites.csv [hunk 1]
```diff
@@ -243,25 +243,35 @@ vc_0337V,https://www.westpalmbeach.va.gov/locations/Jupiter_Vet_Center.asp
...
 vc_0614V,https://www.va.gov/chula-vista-vet-center/
+vc_0624V,https://www.va.gov/san-marcos-vet-center/
 vc_0644V,https://www.va.gov/eureka-vet-center/
 vc_0649V,https://www.va.gov/chico-vet-center/
 vc_0704V,https://www.shreveport.va.gov/locations/Shreveport_La_Veterans_Service_Ctr.asp
 vc_0724V,https://www.va.gov/nashville-vet-center/
 vc_0826MVC,https://www.va.gov/escanaba-vet-center/
+vc_0828MVC,https://www.va.gov/tacoma-vet-center/
+vc_0830MVC,https://www.va.gov/spokane-vet-center/
+vc_08338MVC,https://www.va.gov/las-cruces-vet-center/
```
Comment (as-va) [inline facilities/src/main/resources/websites.csv:268]: This appears likely to be a typo in the source data. All MVCs have 4 digit IDs. `0838MVC` (one three removed) is the listing for "Las Cruces Mobile Vet Center" so it seems likely this is what was intended.

Comment (as-va) [reply facilities/src/main/resources/websites.csv:268]: I'll follow up with Michelle and confirm my hunch with her.

Comment (as-va) [reply facilities/src/main/resources/websites.csv:268]: Just heard back from Michelle: she confirms that ID **was** a typo, so let's go ahead with the correction.
`vc_08338MVC` -> `vc_0838MVC`.

Comment (ChrisBratti-VA) [reply facilities/src/main/resources/websites.csv]: @as-va - I updated `vc_08338MVC` to `vc_0838MVC`, marking as resolved

**Review by as-va — APPROVED** [review]

**Pull request merged**